lib/datasource/go/types.ts

```diff
@@ -3,3 +3,18 @@ export interface DataSource {
   registryUrl?: string;
   lookupName: string;
 }
+
+export interface VersionInfo {
+  Version: string;
+  Time?: string;
+}
+
+export enum GoproxyFallback {
```

Comment:
Can we move the enum to common.ts, so we can import this file as type only. That way the runtime import will be removed.